### ISSUE TITLE
Dynamic UI

### ISSUE THREAD

**Issue by kaus2595** [DSC-Galgotias/Wiki_Scraping#13]
We need to create a dynamic and responsive UI. Current UI was created only for testing purpose.

**Comment by feralCode** [comment]
@kaus2595 can I take this one?

**Comment by kaus2595** [comment]
@ferelcode Yes you can take this one

**Comment by guvi007** [comment]
@kaus2595 Is this issue still open? What do you exactly mean by dynamic and responsive UI? what are the requirements?

**Comment by kaus2595** [comment]
Yes, this issue is still open @guvi007 .Right now the UI we have build is just for test whether the data we want is rendered correctly or not. 
The UI of the application should be more user friendly. For example, where can we put the heading so that it looks awesome to the user.
